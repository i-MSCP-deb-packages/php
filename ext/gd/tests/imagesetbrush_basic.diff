001+ e51a57f0a5f16ab8b7a6366b58ad3375
001- 8168577c0d1fe6d9d11397cb15263d82