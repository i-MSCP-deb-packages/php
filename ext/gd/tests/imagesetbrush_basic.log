
---- EXPECTED OUTPUT
8168577c0d1fe6d9d11397cb15263d82
---- ACTUAL OUTPUT
e51a57f0a5f16ab8b7a6366b58ad3375
---- FAILED
