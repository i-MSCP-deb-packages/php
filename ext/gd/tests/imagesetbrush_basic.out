e51a57f0a5f16ab8b7a6366b58ad3375